packages/kbn-rison/kbn_rison.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// @ts-expect-error untyped module from npm
+// eslint-disable-next-line @kbn/eslint/module_migration
+import Rison from 'rison-node';
+
+export type RisonValue =
+  | boolean
+  | string
+  | number
+  | RisonValue[]
+  | { [key: string]: RisonValue }
+  | null;
+
+export function encodeUnknown(obj: any): string | undefined {
```

Comment:
nit: I wonder if we can replace all the `any`s with `unknown`...

I tested it locally, and `node scripts/type_check.js` pass. Do you think it's worth changing it?

Reply:
IMO it doesn't change anything. Just trying to make the world have a few less `any`s?

Reply:
I think in this case, as an input, any is the most communicative type to use. `unknown` sounds like a mistake to me